.circleci/config.yml

```diff
@@ -12,11 +12,25 @@ workflows:
         - test-2.071-prod
         - test-2.077-upstream-devel
         - test-2.077-upstream-prod
+        - d2tag:
+            context: org-global
+            requires:
+            - test-2.070-devel
+            - test-2.070-prod
+            - test-2.071-devel
+            - test-2.071-prod
+            - test-2.077-upstream-devel
+            - test-2.077-upstream-prod
```

Comment:
I would depend also on D1, do we really want to release a D2 tag if the D1 tag is broken? This would also mean a bit less of special casing.